app/ide-desktop/lib/dashboard/tailwind.config.ts

```diff
@@ -14,7 +14,7 @@ const THIS_PATH = path.resolve(path.dirname(url.fileURLToPath(import.meta.url)))
 
 // The names come from a third-party API and cannot be changed.
 /* eslint-disable no-restricted-syntax, @typescript-eslint/naming-convention */
-export const content = [THIS_PATH + '/src/**/*.tsx']
+export const content = [THIS_PATH + '/src/**/*.tsx', THIS_PATH + '../content/src/**/*.tsx']
```

Comment:
not sure about this tbh... `../content` is a separate JS module so semantically this is wrong...
unfortunately i can't really think of a good solution though. it may be possible to move `panic.tsx` to the dashboard and export it that way, but i'm not sure that's semantically any better

Reply:
Yes, I was also a bit sceptical about this, but I think this is the better of those two easy options. I have cleaned up the path definition a little bit though.